src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/List.js

```diff
@@ -152,7 +154,7 @@ class List extends React.Component<Props> {
         }
 
         if (store !== prevProps.store) {
-            store.updateLoadingStrategy(new this.currentAdapter.LoadingStrategy());
+            store.updateLoadingStrategy(new (this.currentAdapter.getLoadingStrategy())());
```

Comment:
instead of implementing a `getLoadingStrategy()` in each adapter, i would pass the value of the `paginated` prop to the constructor of the `LoadingStrategy`. this also means that we can merge the `FullLoadingStrategy` and `PaginatedLoadingStrategy`